testing-common/src/main/java/io/opentelemetry/instrumentation/testing/util/ContextStorageCloser.java

```diff
@@ -50,13 +51,23 @@ private static void cleanup(AutoCloseable storage) throws Exception {
   private abstract static class ContextRestorer {
     abstract void restore();
 
+    @SuppressWarnings("SystemOut")
     boolean runWithRestore(AutoCloseable target) {
       try {
         target.close();
         return true;
       } catch (Throwable throwable) {
         restore();
         if (throwable instanceof AssertionError) {
+          System.err.println();
```

Comment:
Why to `System.err` and to a logger?

Reply:
`System.err` output felt a bit more readable